dist/index.js

```diff
@@ -74454,20 +74454,31 @@ const packageLockFilename = path.join(
 )
 
 const useYarn = () => fs.existsSync(yarnFilename)
-
 const usePnpm = () => fs.existsSync(pnpmLockFilename)
+const useNpm = () => fs.existsSync(packageLockFilename)
 
 const lockHash = () => {
   const lockFilename = useYarn()
     ? yarnFilename
     : usePnpm()
     ? pnpmLockFilename
-    : packageLockFilename
+    : useNpm()
```

Comment:
more of a nitpick, but I find this ternary a bit difficult to read and would consider refactoring it. But it does work!

Reply:
@AtofStryker 
> more of a nitpick, but I find this ternary a bit difficult to read and would consider refactoring it. But it does work!

That is fair comment. I piggy-backed on the original code and extended it without changing the general syntax. @lmiller1990 had the same comment.

I would plan return to this block of code at some later stage to extend the verification so that it also covers the case of multiple lock files, and that would be a good time to improve the readability.

Thanks for your review!